product-service/serverless.yml

```diff
@@ -24,6 +24,8 @@ functions:
           path: /products
           method: get
           cors: true
+          authorizer:
+            arn: arn:aws:cognito-idp:eu-west-1:583941112965:userpool/eu-west-1_pRzicT5x8
```

Comment:
should arn be in env file? should it be hidden? am I wrong?

Reply:
I'm not specialist in cognito, but what 3rd person can do with arn?
Or what reason to move it to env?